evm/app/plugins.go

```diff
@@ -1,35 +1,31 @@
 package app
 
 import (
+	"fmt"
 	"os"
 
 	"cosmossdk.io/log"
 
 	"pkg.world.dev/world-engine/evm/router"
 	"pkg.world.dev/world-engine/evm/sequencer"
+	"pkg.world.dev/world-engine/rift/credentials"
 )
 
 func (app *App) setPlugins(logger log.Logger) {
-	certPath := os.Getenv("SERVER_CERT_PATH")
-	keyPath := os.Getenv("SERVER_KEY_PATH")
-	if certPath == "" || keyPath == "" {
-		logger.Info("running shard sequencer without SSL certs")
-		app.ShardSequencer = sequencer.NewShardSequencer()
+	routerKey := os.Getenv("ROUTER_KEY")
+	var sequencerOpts []sequencer.Option
+	var routerOpts []router.Option
+	if routerKey == "" {
+		app.Logger().Debug("WARNING: starting the EVM base shard in insecure mode. No ROUTER_KEY provided")
```

Comment:
Why not emit this at the warning level, instead of prefixing this "WARNING" string?

Reply:
cosmos SDK for some reason doesn't have a warning level lol. its just INFO, ERROR, or DEBUG. 🤷🏻